Automatic updating

ItsukiIshizuka opened the pull request:
# What
メッセージの自動更新機能実装

# Why
ユーザインターフェース向上のため

![d5ade3f675797bb7d4713a2fa0193d80](https://user-images.githubusercontent.com/56192097/67376163-a279e680-f5be-11e9-8ace-7fe33c2f53e7.gif)

ItsukiIshizuka committed:
CompleteFunction

ItsukiIshizuka committed:
picture resize

i-date commented on app/assets/javascripts/message.js:
```diff
@@ -50,4 +49,33 @@ $(function() {
+          scroll();
+      })
+      .fail(function() {
+        console.log('error');
```
基本的にconsole.logは開発者しか意識して見ませんので、alertの方がいいでしょう。

i-date reviewed (commented):
ご提出ありがとうございます！コメントいたしました^^

ItsukiIshizuka committed:
ViewCorrection(Not AutomaticUpdating Function)

***これは自動更新機能とは関係ありません***

# What
ビューの修正

#8  Why
グループを画面いっぱいに作成した時にスクロールができないことに気づき、
ビューに修正必要箇所があったので修正しました。

ItsukiIshizuka committed:
JavascriptCorrection

# Javascript
## Correction
(before)
console.log
(after)
alert.log

ItsukiIshizuka commented:
修正しました。
ご確認のよろしくお願いいたします。

i-date commented on app/assets/javascripts/message.js:
```diff
@@ -72,7 +72,7 @@ $(function() {
       })
       .fail(function() {
-        console.log('error');
+        alert.log('error');
```
`alert.log('error')`ではなく、`alert('error')`としましょう。

i-date reviewed (commented):
追加コメントいたしました^^

ItsukiIshizuka committed:
JavascriptCorrection

# Javascript
## Delete
.log

ItsukiIshizuka commented:
修正しました。
中途半端な知識だったため、いい加減な修正をしてしまいました。
今後は確認してから返事します、お手数取らせて申し訳ないです。
よろしくお願いいたします。

shingoteshima commented:
LGTM

ItsukiIshizuka commented:
ご確認いただき、ありがとうございます。
